README.md

```diff
@@ -2,23 +2,22 @@
 
-A mighty, modern CSS linter that helps you enforce consistent conventions and avoid errors in your stylesheets.
+A mighty, modern CSS linter and fixer that helps you avoid errors and enforce consistent conventions in your stylesheets.
 
 ## Features
 
 -   **Over one hundred and fifty rules:** Including those that:
-    -   **Catch errors**: e.g. invalid hex colors, indistinguishable colors, or overriding shorthand properties.
-    -   **Enforce best practices**: e.g. keeping specificity low or disallowing vendor prefixes in your source code.
-    -   **Control what languages features can be used**: e.g. whitelisting specific units, properties and functions, or disallowing certain selector types.
-    -   **Enforce code style conventions**: e.g. checking the spacing around the colon in declarations or specifying patterns for class selectors.
--   **Support for the latest CSS syntax:** Including custom properties, range context for media features, calc() and nesting.
+    -   **Catch errors**: e.g. invalid hex colors, duplicate selectors and overriding shorthand properties.
+    -   **Limit language features**: e.g. vendor prefixes, patterns for selectors and specific units, properties, functions and at-rules.
+    -   **Enforce stylistic conventions**: e.g. the whitespace around the colon in declarations and the number of empty lines between rules.
+-   **Support for the latest CSS syntax:** Including custom properties, calc() and nesting.
 -   **Understands *CSS-like* syntaxes:** The linter is powered by [PostCSS](https://github.com/postcss/postcss), so it understands any syntax that PostCSS can parse, including SCSS, [SugarSS](https://github.com/postcss/sugarss), and *experimental support* for Less.
 -   **Completely unopinionated:** Only enable the rules you want, and configure them with options that tailor the linter to your needs.
 -   **Support for plugins:** It's easy to create your own rules and add them to the linter.
 -   **Automatically fixes some stylistic warnings:** Save time by having stylelint fix your code with this *experimental* feature.
 -   **Shareable configs:** If you don't want to craft your own config, you can extend a shareable config.
 -   **Options validator:** So that you can be confident that your config is valid.
--   **Well tested:** Nearly twenty five thousand tests cover the internals and rules.
+-   **Well tested:** Over ten thousand tests cover the internals and rules.
```

Comment:
I think the way the tests were counted by tape is different to how they are counted now by jest.